src/components/Typography/SwTypography.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <component
+    :is="getComponent"
+    :class="[variant]"
+    :style="{
+      'font-weight': fontWeight,
+      'font-family': `var(--${fontFamily})`,
+    }"
```

Comment:
v-bind in the css is really awesome instead of defining the `style` property, can also fit the `class` property usage